teamB/shinInHuck/myapp/app.js

```diff
@@ -1,50 +1,57 @@
-require('dotenv').config();
-
 const http = require('http');
+
 const express = require('express');
 const cors = require('cors');
 const morgan = require('morgan');
+const dotenv = require('dotenv');
+
+dotenv.config();
```

Comment:
- 인혁님 해당 부분은 아래와 같이 호출과 선언을 바로 하고, 최상단에 작성해주세요 !
```suggestion
require('dotenv').dotenv.config();
```